pipeline-scripts/builds-check.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
note: next time we need to add this somewhere we should add it to a pipeline library, non blocking now